apps/root/webpack.prod.js

```diff
@@ -12,4 +13,50 @@ module.exports = merge(common, {
       'process.env.ARCX_KEY': JSON.stringify(process.env.ARCX_KEY),
     }),
   ],
+  optimization: {
+    minimize: true,
+    minimizer: [
+      // This is only used in production mode
+      new TerserPlugin({
```

Comment:
Have you researched any options to make the prod bundle size smaller?

Reply:
I explored some bundle analyzers and found that a significant portion (around 22 MB) comes from node_modules. To improve this, we should define a new spike in my opinion.
Regarding our app itself, I didn't find any significant areas for improvement during this spike. There are other minimizer plugins available, but I didn't spend time exploring them. 

Something to highlight: Our emojis folder only is 545 Kb, we should keep this in mind for the future

This is our current bundle breakdown for the app, and it's close to what we can expect from it:
<img width="926" alt="Screenshot 2024-07-10 at 2 03 04 PM" src="https://github.com/Balmy-protocol/dca-fe/assets/144699868/86fe154d-d8f6-4e26-bd60-cd21fc266bf8">